### ISSUE TITLE
On command when already on is faulty

### ISSUE THREAD

**Issue by jameswestcott** [kernie66/homebridge-random-delay-switches#7]
This functionality... doesn't work.

"When the delay switch is getting ON command while it's already ON, the timer will restart and the motion sensor trigger will be delayed."

In addition, the next time the switch is turned on, it exhibits random timer durations, even when random is turned off.

**Comment by kernie66** [comment]
Yes, thanks for spotting that. The logic for a restart is missing. I will add that and try to correct the behaviour.
